tests/domains/driving/conftest.py

```diff
@@ -8,7 +8,7 @@
 
 from scenic.domains.driving.roads import Network
 
-mapFolder = Path('tests')/'formats'/'opendrive'/'maps'
+mapFolder = (Path().parent.parent.parent / 'assets' / 'maps').resolve()
```

Comment:
Hmm, this doesn't look right -- the old code seemed to assume the current directory is the root of the repository, but now we're assuming it's the folder containing this file.

Reply:
Good catch, reverted.